BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -288,6 +267,11 @@ SecStartup (
     Status = PeCoffRelocateImage (StageHdr->Base);
     if (!EFI_ERROR (Status)) {
       EnableCodeExecution ();
+
+      ImageContext.ImageAddress = (PHYSICAL_ADDRESS)(UINTN) StageHdr->Base;
```

Comment:
Can we move PeCoffLoaderGetPdbPointer and PeCoffLoaderRelocationImageExtraAction into  PeCoffRelocationImage() ?

Reply:
Okay, I will do that.